lib/api.js

```diff
@@ -8,40 +8,72 @@ var http = require('http');
 var https = require('https');
 var stringify = require('json-stringify-safe');
 
-exports.VERSION = '1';
-exports.endpoint = 'https://api.rollbar.com/api/' + exports.VERSION + '/';
-exports.accessToken = null;
-
-
 var SETTINGS = {
-  accessToken: null,
   protocol: 'https',
-  endpoint: exports.endpoint,
+  version: '1',
+  endpoint: 'https://api.rollbar.com/api/1/',
```

Comment:
I'd like to keep `version` and `endpoint` as globals so we can quickly identify which version is being used. Also, we need to generate the endpoint using the version.